lib/matplotlib/axes/_axes.py

```diff
@@ -1288,132 +1288,228 @@ def eventplot(self, positions, orientation='horizontal', lineoffsets=1,
     @docstring.dedent_interpd
     def plot(self, *args, **kwargs):
         """
-        Plot lines and/or markers to the
-        :class:`~matplotlib.axes.Axes`.  *args* is a variable length
-        argument, allowing for multiple *x*, *y* pairs with an
-        optional format string.  For example, each of the following is
-        legal::
+        Plot lines and/or a series of markers. This is also known as line plot
+        or XY (scatter) plot respectively.
```

Comment:
Using the word 'scatter' here dangerous as we also have a `scatter` method.  I worry about compounding the confusion.

Reply:
Unfortunately, the confusion is inherent in the naming of the methods. The general notion of [scatter plot](https://en.wikipedia.org/wiki/Scatter_plot) in its basic form is what we're doing here. That's why I added the sentence. I've also added the scatter method in the *See Also* section.

IMO doing both is as good as we can get to prevent confusion. If you insist, one could add an additional sentence: "Note that matplotlib also offers a scatter method which can additionally change marker properties per data."

Reply:
I don't think you need the use of "scatter" in here either.  "Plot *y* versus *x* as lines and/or markers" is how I'd phrase this.